catroid/src/main/java/org/catrobat/catroid/common/BrickValues.java

```diff
@@ -180,6 +180,7 @@
 	public static final int STITCH_LENGTH = 10;
 	public static final int ZIGZAG_STITCH_LENGTH = 2;
 	public static final int ZIGZAG_STITCH_WIDTH = 10;
+	public static final String THREAD_COLOR = "0xff0000";
```

Comment:
```suggestion
	public static final String THREAD_COLOR = "#ff0000";
```
We use the #XXXXXX format for colors in Catrobat, see all other color related bricks, and that's also what the color picker returns.